provisioner.yaml

```diff
@@ -0,0 +1,35 @@
+# Score provides a developer-centric and platform-agnostic 
```

Comment:
I think @rabelmervin you should not add this cause it's irrelevant to issue #220 
Also, it is of no use.